pandas/core/series.py

```diff
@@ -1002,36 +1002,88 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Generate a new DataFrame with columns containing the data and 
+        the index of the original Series, or, if drop is True, 
+        a Series with numeric index.
+        
+        For an Index Series, the results is a two-column DataFrame with 
+        index tourned in column.
+        For a MultiIndex Series, the results is a multi-column 
+        DataFrame with each level being turned into a column.
+        The new columns will be named level_n (with n increasing frm 0) 
+        if the name is None. 
 
         Parameters
         ----------
         level : int, str, tuple, or list, default None
             Only remove the given levels from the index. Removes all levels by
-            default
-        drop : boolean, default False
-            Do not try to insert index into dataframe columns
+            default.
+        drop : bool, default False
+            Do not try to insert index into dataframe columns.
         name : object, default None
-            The name of the column corresponding to the Series values
-        inplace : boolean, default False
-            Modify the Series in place (do not create a new object)
+            The name of the column corresponding to the Series values.
+        inplace : `bool`, default False
+            Modify the Series in place (do not create a new object).
 
         Returns
         ----------
-        resetted : DataFrame, or Series if drop == True
+        reset : DataFrame, or Series if drop == True
```

Comment:
Check the first example here to see how the output of Returns is: https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-4-returns-or-yields

Also, not your change, but the hyphens under return are too long.